src/item.ts

```diff
@@ -32,7 +34,8 @@ const enum ElementKind {
     Instagram,
     Audio,
     Embed,
-    Video
+    Video,
+    ContentAtom
```

Comment:
What's the difference between an "atom" and a "content atom"?

Reply:
the block type is `ContentAtom` in capi. Maybe we could name it `Atom`

Reply:
We seem to refer to it mostly as "atom" so `Atom` sounds good.